mediator/src/controllers/endpoint.ts

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+import { FHIR } from "../../config";
+import { logger } from "../../logger";
+
+const { url: fhirUrl, username, password } = FHIR;
```

Comment:
In `encounter.ts` you use: 

`const { url: fhirUrl, username: fhirUsername, password: fhirPassword } = FHIR;`

Please be consistent and choose one approach.